### ISSUE TITLE
Sidebar menu component for events

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#135]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue is currently blocked by #85, but I'll be working on this shortly. We'd like to take the base component being worked on in #85 and add the parts of it that will be inserted for event pages. This component can be seen in the [designs in Figma](https://www.figma.com/file/I9McFfaLu1RiiWp5IP3YjE/activist_public_designs?type=design&node-id=766%3A214&t=2a7k5T7ji0MelYTs-1) and further is shown below:

![Section Selectors](https://github.com/activist-org/activist/assets/24387426/77a086cc-37e3-4c7d-8e19-725ce7c11042)

Note that there shouldn't be a background for the component and it should be transparent over the sidebar background.

### Contribution

Will work on this myself eventually or would be glad to support someone who wants to work on it themselves :)

**Comment by andrewtavis** [comment]
Assigning myself to hold this issue for the team (of one 👏) working on it for Code Night 🌙😊

**Comment by andrewtavis** [comment]
I'll also make a component for the event icon that has the type color indicator on the left :)

**Comment by andrewtavis** [comment]
@Masa-99, FYI there's a component for the event icon now as seen on the [home page](https://activist.org/home/). Basically fills whatever `div` it's put into with the intent being that we'll put it into square containers :)

Feel free to write in here and in #134 so I can assign 🙃

**Comment by Masa-99** [comment]
Thank for the update! :) Happy to implement this and #134

**Comment by andrewtavis** [comment]
Assigned! :) Let me know if I can help. Looking forward to checking out some of your techniques and fixing some things (like component images 🤦‍♂️) 😊

**Comment by andrewtavis** [comment]
Closed via bf029de 😊 Thank you, @Masa-99! Let me know if and when you have time for another issue!

**Comment by andrewtavis** [comment]
Hey @Masa-99! FYI we're doing a call on Saturday to discuss the backend, if you're interested/available :) Time is gonna be 3pm CEST. Details for in are in [the development room of our matrix community that I talked to you about and is a super cool new messaging tool unlike any of the other "super cool messaging tools" from before](https://matrix.to/#/!CRgLpGeOBNwxYCtqmK:matrix.org?via=matrix.org&via=acter.global&via=chat.0x7cd.xyz) 🙃

Would be great if you could attend!
